base_domain_operator/tests/test_base_domain_operator.py

```diff
@@ -2,9 +2,14 @@
 # © 2017 Therp BV <http://therp.nl>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 from openerp.tests.common import TransactionCase
+from ..expression import Expression
 
 
 class TestBaseDomainOperator(TransactionCase):
+
+    post_install=True
+    at_install=False
```

Comment:
this will fail flake8

Reply:
PS: install the OCA precommit hook https://github.com/OCA/maintainer-quality-tools/tree/master/git so that you get those checks before committing